test/e2e/node/node_problem_detector.go

```diff
@@ -239,7 +239,7 @@ func verifyNodeCondition(f *framework.Framework, condition v1.NodeConditionType,
 	return nil
```

Comment:
Another example can be : 
https://github.com/kubernetes/kubernetes/blob/c5cc25d1cb075f84756db9bffecd3fafe12c5fab/test/e2e/node/node_problem_detector.go#L228